src/components/Swap/SwapButton/index.tsx

```diff
@@ -84,16 +89,13 @@ export default memo(function SwapButton({ disabled }: SwapButtonProps) {
       })
       setDisplayTxHash(transaction.hash)
     } catch (e) {
+      postWrap()
```

Comment:
This will occur before the event listener is added.

Why doesn't postWrap cover the case of a failed transaction already, without this PR?